_std/defines/projectiles.dm

```diff
@@ -45,8 +45,10 @@
 
 #define AMMO_PISTOL_22 "pistol_22"
 #define AMMO_PISTOL_9MM "pistol_9mm"
+#define AMMO_PISTOL_9MM_SOVIET "pistol_9mm_soviet"
+#define AMMO_PISTOL_9MM_ALL AMMO_PISTOL_9MM, AMMO_PISTOL_9MM_SOVIET
 #define AMMO_PISTOL_41 "pistol_41"
-#define AMMO_PISTOL_ALL AMMO_PISTOL_22, AMMO_PISTOL_9MM, AMMO_PISTOL_41
+#define AMMO_PISTOL_ALL AMMO_PISTOL_22, AMMO_PISTOL_9MM, AMMO_PISTOL_9MM_SOVIET, AMMO_PISTOL_41
```

Comment:
Synced with @virvatuli-gh offline. Another requested change was that the makarov couldn't accept any other 9mm ammo, but allow the reverse to still be possible.

Created a separate ammo cat (🐈) for this case.